webui/module/Auth/src/Auth/Controller/AuthController.php

```diff
@@ -273,4 +222,58 @@ private function createNewLoginForm($form, $multi_dird_env = null, $err_msg = nu
          )
       );
    }
+
+   /**
+    * DIRD update check
+    */
+   private function checkUpdateStatusDIRD() {
+      try {
+         $dird_version = $this->getDirectorModel()->getDirectorVersion($this->bsock);
+      }
+      catch(Exception $e) {
+         echo $e->getMessage();
```

Comment:
This `catch`-block should set `$dird_version` to a sane value as it is used later, or it should cancel the execution of the function. 
In the latter case it might be beneficial to remove the `try`-block, as there is usually no point in catching an exception you cannot handle.

Reply:
Adjusted.